apps/spruce/src/pages/adminSettings/index.tsx

```diff
@@ -196,111 +196,172 @@ const AdminSettingsPage: React.FC = () => {
                 Disabled GraphQL Queries
               </SideNavItem>
             </SideNavGroup>
-
-            <SideNavGroup header="Authentication">
+            <SideNavGroup header="External Communication">
```

Comment:
[nit] The title on the page says "External Communications"

```suggestion
            <SideNavGroup header="External Communications">
```